### DISCUSSION TITLE
Why isn't "remove HI" catching words before colons?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5130] [General]
![image](https://user-images.githubusercontent.com/82276985/122434451-1cfe3380-cf65-11eb-90e2-8176dcc107bf.png)

**Comment by niksedk** [comment]
Could you attach the subtitle - always make things much easier, thx :)

**Reply by DrReddP** [reply]
[Before removing HI.txt](https://github.com/SubtitleEdit/subtitleedit/files/6671884/Before.removing.HI.txt)
[After Removing HI.txt](https://github.com/SubtitleEdit/subtitleedit/files/6671886/After.Removing.HI.txt)

**Comment by niksedk** [comment]
The first line is missing a sentence terminator, like period, exclamation mark, or question mark:
````` 
<i>- You're my only friend, Abu
- WOMEN: Who?</i>`

**Reply by DrReddP** [reply]
It's after a dialogue marker, though

**Reply by niksedk** [reply]
A dash can also be a pause - or a line continuation, or even a person stopping talking abruptly or other stuff.
Garbage in = garbage out :)
